api/src/pcapi/core/mails/transactional/bookings/common.py

```diff
@@ -0,0 +1,34 @@
+from pcapi.core.bookings.models import Booking
+from pcapi.utils.date import format_time_in_second_to_human_readable
+
+
+def get_booking_token(booking: Booking) -> str:
+    return booking.activationCode.code if booking.activationCode else booking.token
+
+
+def get_offer_withdrawal_delay(booking: Booking) -> str | None:
+    return (
+        format_time_in_second_to_human_readable(booking.stock.offer.withdrawalDelay)
+        if booking.stock.offer.withdrawalDelay
+        else None
+    )
+
+
+def get_offer_withdrawal_details(booking: Booking) -> str | None:
+    return booking.stock.offer.withdrawalDetails or None
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Le `or None` convertit `None` en `None` ? Ou bien il faut gérer `""` ?
```suggestion
    return booking.stock.offer.withdrawalDetails
```

Reply:
j'imagine qu'on est pas censé avoir de chaine vide, mais au cas ou, je pense que c'est mieux de renvoyer `None` dans tous les cas ou on n'a pas l'info.